src/HTML.js

```diff
@@ -261,7 +261,7 @@ export default class HTML extends PureComponent {
                 children = alteredChildren || children;
             }
             // Remove whitespaces to check if it's just a blank text
-            const strippedData = data && data.replace(/\s/g, '');
+            const strippedData = data && data.replace(/\s+/g, ' ');
```

Comment:
In the example you provided:
```html
Some <b>bold</b> text.
```
`strippedData` value would be ` text` while previously it would have been `text`. But `strippedData` is not the rendered text, it is solely used as a check to bypass rendering for blank text nodes.

Reply:
However this change would be relevant for this kind of example:
```html
<strong><em>Hello</em></strong> <strong>World</strong>
```
@callaars could you explain your rationale in the example you provided?